ptable/ptable.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Partition table
+ *
+ * Copyright 2020 Phoenix Systems
+ * Author: Hubert Buczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#ifndef _LIB_PARTITION_TABLE_H_
+#define _LIB_PARTITION_TABLE_H_
+
+
+/* Structure of partition table:
```

Comment:
please, add info it the values would be big endian or little endian (we will be creating partition table on host system and reading it on the device - the host tool has to take endianness into account).